dev/rename_docker_images_to_main.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import subprocess
+from typing import List
+
+PYTHON_VERSIONS = ["3.6", "3.7", "3.8"]
+
+DOCKERHUB_PREFIX = "apache/airflow"
+
+DOCKERHUB_IMAGES = [
+    "{prefix}:python{python_version}-{branch}",
```

Comment:
Is image tag purposely ignored for `DOCKERHUB_IMAGES`?

I notice they are all `latest` for `GITHUB_REGISTRY_IMAGES` and `GHCR_IO_IMAGES`.

Reply:
The problem with DOCKERHUB_IMAGES is that they are all "apache/airflow" - because that's the only image we have access to in DockerHub. So in DockerHub the image is always "apache/airflow" but has a different tag. 

In GitHub registries (depends on the type of registry) we have more flexibility":
* in the old GitHub docker registry - docker.pkg.github.com -  (current but already deprecated) we can use "apache/airflow/IMAGE:tag"
* in the new package registry (ghcr.io) - we can submit anything under apache/airflow-* but then we link it to the project via docker image label.

So naming conventions for all three types of images are different :(